src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -175,7 +174,7 @@ export default class NameTranslation extends Mixins(CommonMixin) {
   private get hasPendingChange (): boolean {
     return this.draftTranslations.length !== this.nameTranslations.length ||
       !this.draftTranslations.every((translation, index) => {
```

Comment:
This could be improved by changing `!every(x===y)` to `some(x!==y)` :)

If you don't feel like making this change, we'll get to it later.